packages/console/src/pages/Cloud/pages/Congrats/index.module.scss

```diff
@@ -0,0 +1,53 @@
+@use '@/scss/underscore' as _;
+
+.content {
+  align-items: center;
+}
+
+.title {
+  margin-top: _.unit(6);
+  font: var(--font-title-2);
+  text-align: center;
+}
+
+.description {
+  margin: _.unit(1) 0 _.unit(6) 0;
+  font: var(--font-body-2);
+  color: var(--color-text-secondary);
+}
+
+.divider {
+  height: 0;
+  width: 100%;
+  border-top: 1px solid var(--color-divider);
+  margin: _.unit(8) 0;
```

Comment:
nit: If you decide to extract divider as a component, remember do not include margin in the component scss